demos/asr-automation-recovery/azuredeploy.parameters.json

```diff
@@ -7,6 +7,9 @@
     },
     "automationRegion": {
       "value": "westeurope"
+    },
+    "modulesUri": {
+        "value": "https://devopsgallerystorage.blob.core.windows.net/packages/"
```

Comment:
if this is required, then set this as the defaultValue in the template and remove this from here

Reply:
If we set this as the default value in the template, the TTK throws best practice violation error for hard coded url. Should we do it still?

Reply:
Updated, but fails BP validation.